freqtrade/exchange/exchange.py

```diff
@@ -1486,12 +1510,25 @@ def get_historic_trades(self, pair: str,
         :returns List of trade data
         """
         if not self.exchange_has("fetchTrades"):
-            raise OperationalException("This exchange does not suport downloading Trades.")
+            raise OperationalException("This exchange does not support downloading Trades.")
 
         return asyncio.get_event_loop().run_until_complete(
             self._async_get_trade_history(pair=pair, since=since,
                                           until=until, from_id=from_id))
 
+    def get_interest_rate(self, pair: str, open_rate: float, is_short: bool) -> float:
+        # TODO-lev: implement
+        return 0.0005
+
+    def set_leverage(self, pair, leverage):
+        """
+            Binance Futures must set the leverage before making a futures trade, in order to not
+            have the same leverage on every trade
+            # TODO-lev: This may be the case for any futures exchange, or even margin trading on
+            # TODO-lev: some exchanges, so check this
+        """
+        self._api.set_leverage(symbol=pair, leverage=leverage)
```

Comment:
We can't for sure call this here - but MUST implement it on exchanges that require it.

calling this on exchanges that don't support it will fail - so IF it generalizes (is needed by more than binance) - it must be an empty method - not one that's calling a ccxt method.